.circleci/config.yml

```diff
@@ -111,10 +76,6 @@ jobs:
       - install-ci-pip-deps
       - do-static-analysis
       - non-coverage-tests
-      - when:
-          condition: << pipeline.parameters.run_coverage >>
```

Comment:
There is a trigger defined in the CircleCI system. The trigger defines this parameter. Should we just change that and keep the authoring here?